src/server.cpp

```diff
@@ -1958,6 +1976,30 @@ void Server::SendMovePlayer(u16 peer_id)
 	Send(&pkt);
 }
 
+void Server::SendKnockBack(const u16 peer_id, const v3f &direction, const f32 &time_knockback, const u16 id_player_knockback)
```

Comment:
since it's not a reference or pointer changing the variable would have no effect on the outside of the function anyway -> makes no sense